src/app/routes/loadInitialData/index.js

```diff
@@ -11,7 +12,10 @@ const loadInitialData = async (url, routes) => {
     return {};
   }
 
-  return route.getInitialData({ match });
+  const data = await route.getInitialData({ match });
+
+  data.dials = getDials();
```

Comment:
The work that chris just did to split the data object can be expanded here. Plus, loadInitialData is a function thats also used by the client. I know you have
```
const dialsPath = process.env.COSMOS_DIALS_PATH;

if (!dialsPath) {
 return {};
}
```
Which in theory should stop the `fs.readFileSync` from running on the client, but we shouldn't risk it.
Can you move this function call to [here](https://github.com/bbc/simorgh/blob/latest/src/server/index.jsx#L144) and pass the data through like BBCOrigin please? This avoids any risk of this ever been called on the client